printer/colors_definition.go

```diff
@@ -38,3 +38,51 @@ var (
 	HeaderColorForLight   = color.Black // for plain table
 	RequiredColorForLight = color.Red   // for `kubectl explain`
 )
+
+type ColorSchema struct {
+	DefaultColor, // defaut when no specific mapping is found for the command
+	KeyColor, // used to color keys (where ?)
+	StringColor, // default color for strings
+	TrueColor, // used when value is true
+	FalseColor, // used when value is false or an error
+	NumberColor, // used when the value is a number
+	NullColor, // used when the value is null or a warning
+	HeaderColor, // used to print headers
+	FreshColor, // color used when the time value is under a certain delay
+	RequiredColor color.Color // used when the value is required or is an error
+	RandomColor []color.Color // used to display multiple colons, cycle between colors
```

Comment:
Suggest also adding configs for status colors.

Such as a StatusOK, StatusWarning, and StatusError.

If we want statuses to be configurable too later, then we could just let the user redefine which status texts belong to which status. Not saying we should add that now, but just pointing out that we're not painting ourselves into a corner at least.